service/resource/configmap/error.go

```diff
@@ -31,3 +31,31 @@ var invalidConfigMapError = microerror.New("invalid config map")
 func IsInvalidConfigMap(err error) bool {
 	return microerror.Cause(err) == invalidConfigMapError
 }
+
+var configMapAssertionError = microerror.New("configmap assertion")
+
+// IsConfigMapAssertion asserts configMapAssertionError.
+func IsConfigMapAssertion(err error) bool {
```

Comment:
The naming still stutters IMO. Also what is the error with a configmap assertion? The error message does not reflect an obvious problem I can comprehend from the top of my head.

Reply:
We apparently don't even need this anymore. Removed.